README.md

```diff
@@ -15,26 +15,75 @@ Once you have implemented the `sa` program (and tested it to the best of your ab
 
 # Report
 
-Jeg forventer at blive færdig med rapporten i dag (lørdag) <br>
--Gaia
 
+## Algorithm
 
+We used the construction algorithm based on raddix sort. 
+The algorithm has a running time of O(n²), which we will compare to the running time of our implementation below. 
 
-## Algorithm
+## Insights you may have had while implementing the algorithm
+I [Gaia] have learned a lot about C and how it compares to Python. 
+Since I started writing the algorithms in Python, the differences became clear, 
+when translated to C. 
+Some consepts I have a better understanding of include pointers, 
+structs, iterator pattern, header files, char as int, and strings. <br>
+Some insights regarding Windows vs. Linux became clear, when we experienced problem, as described below. 
```

Comment:
It takes some getting used to, yes, when coming from a high-level language like Python.